console/console-init/ui/src/Routes.tsx

```diff
@@ -18,8 +18,8 @@ const getConnectionDetail = () =>
   import("modules/connection-detail/ConnectionDetailPage");
 const getDeviceDetailPage = () =>
   import("modules/iot-device-detail/DeviceDetailPage");
-const getIoTProjectDetailPage = () =>
-  import("modules/iot-project-detail/IoTProjectDetailPage");
+const getProjectDetailPage = () =>
```

Comment:
I will suggest name as getIoTProjectDetailPage as it will create confusion in messaging and iot projects